grr/server/grr_response_server/gui/ui/components/client_add_label_dialog/client_add_label_dialog.ts

```diff
@@ -0,0 +1,97 @@
+import {Component, OnInit, Inject} from '@angular/core';
+import {MatDialogRef, MAT_DIALOG_DATA} from '@angular/material/dialog';
+import {FormControl, ValidatorFn, AbstractControl} from '@angular/forms';
+import {Observable} from 'rxjs';
+import {startWith, map, combineLatest, filter} from 'rxjs/operators';
+import {ClientLabel} from '@app/lib/models/client';
+import {ConfigFacade} from '@app/store/config_facade';
+
+@Component({
+  selector: 'client-add-label-dialog',
+  templateUrl: './client_add_label_dialog.ng.html',
+  styleUrls: ['./client_add_label_dialog.scss'],
+})
+export class ClientAddLabelDialog {
+  constructor(
+    readonly dialogRef: MatDialogRef<ClientAddLabelDialog>,
+    @Inject(MAT_DIALOG_DATA) private clientLabels: ReadonlyArray<ClientLabel>,
```

Comment:
Is it possible to change both to `private readonly`?

Reply:
Totally